content/docs/api-reference/get_url.md

```diff
@@ -31,22 +31,25 @@ specified by its `path` in a `repo` (<abbr>DVC project</abbr>), is stored.
 The URL is formed by reading the project's
 [remote configuration](/doc/command-reference/config#remote) and the `dvc.yaml`
 or `.dvc` file where the given `path` is found (`outs` field). The schema of the
-URL returned depends on the
-[type](/doc/command-reference/remote/add#supported-storage-types) of the
-`remote` used (see the [Parameters](#parameters) section).
+URL returned depends on the [type][storage-types] of the `remote` used (see the
+[Parameters](#parameters) section).
 
 If the target is a directory, the returned URL will end in `.dir`. Refer to
-[Structure of cache directory](/doc/user-guide/project-structure/internal-files#structure-of-the-cache-directory)
-and `dvc add` to learn more about how DVC handles data directories.
+[Structure of cache directory] and `dvc add` to learn more about how DVC handles
+data directories.
 
 ⚠️ This function does not check for the actual existence of the file or
 directory in the remote storage.
 
 💡 Having the resource's URL, it should be possible to download it directly with
-an appropriate library, such as
-[`boto3`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Object.download_fileobj)
-or
-[`paramiko`](https://docs.paramiko.org/en/stable/api/sftp.html#paramiko.sftp_client.SFTPClient.get).
+an appropriate library, such as [`boto3`] or [`paramiko`].
+
+[storage-types]: /doc/command-reference/remote/bad
+[structure of cache directory]:
+  /doc/user-guide/project-structure/internal-files#bad
+[`boto3`]: https://bad
+[`paramiko`]:
+  https://bad.paramiko.org/en/stable/api/sftp.html#paramiko.sftp_client.SFTPClient.get
```

Comment:
Added these to test the updated link checker now that https://github.com/iterative/link-check/pull/6 is merged.

Reply:
Perfect! https://github.com/iterative/dvc.org/pull/2534/checks?check_run_id=2969451349
![image](https://user-images.githubusercontent.com/1477535/124225935-e3353d00-dacd-11eb-81a4-ee15a8ac1cc4.png)

Reply:
(Fixed bad links.)